src/server/game/Spells/Spell.cpp

```diff
@@ -5831,46 +5831,69 @@ SpellCastResult Spell::CheckRange(bool strict)
     if (!strict && m_casttime == 0)
```

Comment:
@Shauren its true, but target can be NULL. When target == m_caster and !target -- two different cases? Peace of code where ```if (m_targets.HasDst() && !m_targets.HasTraj())``` is not needed if target == m_caster?